pkg/odo/genericclioptions/context.go

```diff
@@ -72,6 +72,42 @@ func checkProjectCreateOrDeleteOnlyOnInvalidNamespace(command *cobra.Command, er
 	}
 }
 
+func checkValidConfig(command *cobra.Command, configFileName string) {
```

Comment:
error should be returned

Reply:
Did the best i could for now, it is returned and handled by caller. But will require bigger refactor